documentation/staging/content/samples/simple/domains/domain-home-in-image/_index.md

```diff
@@ -86,16 +113,16 @@ The usage of the create script is as follows:
 ```shell
 $ sh create-domain.sh -h
 ```
-```
-usage: create-domain.sh -o dir -i file -u username -p password [-k] [-e] [-h]
+```text
+usage: create-domain.sh -o dir -i file -u username -p password [-m wdt|wlst] [-n encryption-key] [-e] [-v] [-h]
   -i Parameter inputs file, must be specified.
   -o Ouput directory for the generated properties and YAML files, must be specified.
   -u User name used in building the image for WebLogic domain in image.
   -p Password used in building the image for WebLogic domain in image.
```

Comment:
I realize this is old wording, but consider rewording to something that more clearly states what the user/pass is meant for: Something like: "WebLogic administrator user name for the WebLogic domain" and "WebLogic administrator password for the WebLogic domain".